lib/compat/wordpress-6.9/script-modules.php

```diff
@@ -1,91 +0,0 @@
-<?php
-/**
- * Script Modules compatibility functions for WordPress 6.9.
- *
- * Patches WP_Script_Modules to add support for the wp_script_module_attributes filter
- * which is intended to be backported to core.
- *
- * @package gutenberg
- */
-
-if ( version_compare( get_bloginfo( 'version' ), '6.9.0', '<' ) ) {
-
-	remove_action( 'after_setup_theme', array( wp_script_modules(), 'add_hooks' ) );
-	add_action( 'after_setup_theme', 'gutenberg_script_modules_add_hooks', 9 );
-
-	/**
-	 * Adds hooks for script modules with custom printing logic.
-	 *
-	 * Replaces the default script module hooks with custom ones that support
-	 * filtering script module attributes via the wp_script_module_attributes filter.
-	 */
-	function gutenberg_script_modules_add_hooks() {
-		$wp_script_modules = wp_script_modules();
-		$position          = wp_is_block_theme() ? 'wp_head' : 'wp_footer';
-
-		add_action( $position, array( $wp_script_modules, 'print_import_map' ) );
-		add_action( $position, 'gutenberg_print_enqueued_script_modules' );
-		add_action( $position, array( $wp_script_modules, 'print_script_module_preloads' ) );
-
-		add_action( 'admin_print_footer_scripts', array( $wp_script_modules, 'print_import_map' ) );
-		add_action( 'admin_print_footer_scripts', 'gutenberg_print_enqueued_script_modules' );
-		add_action( 'admin_print_footer_scripts', array( $wp_script_modules, 'print_script_module_preloads' ) );
-
-		add_action( 'wp_footer', array( $wp_script_modules, 'print_script_module_data' ) );
-		add_action( 'admin_print_footer_scripts', array( $wp_script_modules, 'print_script_module_data' ) );
-		add_action( 'wp_footer', array( $wp_script_modules, 'print_a11y_script_module_html' ), 20 );
-		add_action( 'admin_print_footer_scripts', array( $wp_script_modules, 'print_a11y_script_module_html' ), 20 );
-	}
-
-	/**
-	 * Prints enqueued script modules with support for the wp_script_module_attributes filter.
-	 *
-	 * Uses reflection to access private methods of WP_Script_Modules and prints
-	 * script module tags with attributes that can be filtered via the
-	 * wp_script_module_attributes filter.
-	 */
-	function gutenberg_print_enqueued_script_modules() {
-		$wp_script_modules = wp_script_modules();
-		$reflection        = new ReflectionClass( $wp_script_modules );
-
-		// Gets private methods via reflection.
-		$get_marked_for_enqueue = $reflection->getMethod( 'get_marked_for_enqueue' );
-		$get_marked_for_enqueue->setAccessible( true );
```

Comment:
Aside: this is causing a PHP 8.5 deprecation warning:

> Deprecated: Method ReflectionMethod::setAccessible() is deprecated since 8.5, as it has no effect in /var/www/src/wp-content/plugins/gutenberg/lib/compat/wordpress-6.9/script-modules.php on line 53

Reply:
Isn't that file removed in this PR?

Reply:
Yes, but I'm just saying this adds to the importance of getting this merged.